bespokefit_tutorial/bespoke_parameters_showcase.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
<p>Would it be worth putting a sub-title over this? i.e. "Reading input files"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yeah good idea</p>